### PR TITLE
Fix find similar incidents - ignore future incidents

### PR DESCRIPTION
**Pull request by erezh31**

### CODE REVIEW

**Comment by erezh31** [comment]
fixes https://github.com/demisto/etc/issues/16109

**Comment by meirwah** [comment]
@erezh31 can you fix test?
image

File: Scripts/script-FindSimilarIncidents.yml [hunk 1]
```diff
@@ -142,7 +142,7 @@ script: |-
       condition_string = ' %s ' % applied_condition.lower()
       similar_keys_query = condition_string.join(map(lambda t: '%s:"%s"' % (t[0], escape_special_query_chars(t[1])), similar_incident_keys.items()))
       incident_time = parse_datetime(incident_time)
-      max_date = incident_time + timedelta(hours=hours_back)
+      max_date = incident_time
```
Comment (meirwah) [inline Scripts/script-FindSimilarIncidents.yml]: why not make this an argument?

Comment (erezh31) [reply Scripts/script-FindSimilarIncidents.yml]: I can, not sure it's correct logic - we want to prevent "loop" option they talked about it in the issue

Comment (meirwah) [reply Scripts/script-FindSimilarIncidents.yml]: ok, then nevermind.. let's fix tests and merge

Comment (erezh31) [reply Scripts/script-FindSimilarIncidents.yml]: will fix

**Pull request merged**